src/Refitter.Tests/SwaggerPetstoreTests.cs

```diff
@@ -414,6 +414,25 @@ public async Task Can_Build_Generated_Code_With_IApiResponse(SampleOpenSpecifica
             .BeTrue();
     }
 
+    [Theory]
+    [InlineData(SampleOpenSpecifications.SwaggerPetstoreJsonV3, "SwaggerPetstore.json")]
+    public async Task Can_Build_Generated_Code_With_IObservableResponse(SampleOpenSpecifications version, string filename)
+    {
+        var settings = new RefitGeneratorSettings();
+        settings.ReturnIObservable = true;
+        var generateCode = await GenerateCode(version, filename, settings);
+        //cannot build without it because System.Reactive package has to be installed first
+        generateCode += @"
```

Comment:
You can add the `System.Reactive` package reference to the [ProjectFileContents.cs](https://github.com/christianhelle/refitter/blob/main/src/Refitter.Tests/Build/ProjectFileContents.cs) file

```xml
<PackageReference Include=""System.Reactive"" Version=""6.0.0"" />
```

so you don't need to insert a fake `Unit` type